spec/classes/rabbitmq_spec.rb

```diff
@@ -1384,6 +1384,17 @@
     end
   end
 
+  context "on FreeBSD" do
+    with_freebsd_facts
+    it 'installs the rabbitmq package' do
+      should contain_package('rabbitmq-server').with(
+        'ensure'   => 'installed',
+        'name'     => 'rabbitmq',
+        'provider' => 'freebsd'
```

Comment:
This is why your test is failing, it's expecting pkgng.
Though I think provider isn't actually needed. Not sure if #493 will get merged, but would have to adjust slightly to get FreeBSD support working if / when that gets merged.